homeassistant/components/bluesound/media_player.py

```diff
@@ -4,6 +4,9 @@
 from datetime import timedelta
 import logging
 
```

Comment:
Please remove this blank line so that urllib is grouped with the other standard library imports.